docs/connectors/postgresql.asciidoc

```diff
@@ -61,7 +61,8 @@ Note that *if your database is hosted by a service* such as https://www.heroku.c
 It is possible to monitor PostgreSQL database running in https://aws.amazon.com/rds/[Amazon RDS]. To get it running you must fulfill the following conditions
 
 * The instance parameter `rds.logical_replication` is set to `1`.
-* Verify that `wal_level` parameter is set to `logical`; this might not be the case in multi-zone replication setups.
+* Verify that `wal_level` parameter is set to `logical`; this might not be the case in multi-zone replication setups, you cannot set this option manually, it's (https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.html[automatically changed]) when the `rds.logical_replication` is set to `1`. You can verify it by running the query `SHOW wal_level` as DB master user.
```

Comment:
Please write a single sentence per-line if possible

Reply:
I tried to simplify it a bit, lmk what you think!